src/state/pipeline-runs/index.js

```diff
@@ -0,0 +1,6 @@
+import { makeLocalGetter } from '../../utils/object';
+import get from 'lodash/get';
+
+const localGetter = makeLocalGetter('pipelineRuns');
+
+export const getPipelineRuns = (state) => get(state, 'pipelineRuns', []);
```

Comment:
remove unused getter
```suggestion
import get from 'lodash/get';

export const getPipelineRuns = (state) => get(state, 'pipelineRuns', []);
```